package.json

```diff
@@ -875,12 +875,12 @@
         },
         {
           "command": "confluent.diff.selectForCompare",
-          "when": "view == confluent-schemas && viewItem =~ /.*-schema$/",
+          "when": "(view == confluent-schemas || view == confluent-topics) && viewItem =~ /.*-schema$/",
```

Comment:
Unrelated to this PR: I don't know if we never supported diffing schema versions from the Topics view, but it seemed weird that it was missing when I was click-testing.